tests/integration/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/skip-mev/connect/v2/tests/integration
```

Comment:
So basically when it is like this, we are referring to the go.mod file inside connect